pkg/apis/v1alpha1/condition.go

```diff
@@ -0,0 +1,22 @@
+package v1alpha1
+
+import corev1 "k8s.io/api/core/v1"
+
+type ConditionType string
+
+const (
+	Ready    ConditionType = "Ready"
+	NotReady ConditionType = "NotReady"
+)
+
+// Condition defines condition of custom resource.
+type Condition struct {
+	// Type defines condition type.
+	Type ConditionType `json:"type"`
+	// Status defines status of condition type, True or False.
```

Comment:
Status of the condition, one of True, False, Unknown.

Reply:
Thanks, fixed.